react/features/chat/actionTypes.js

```diff
@@ -13,6 +13,16 @@
  */
 export const ADD_MESSAGE = Symbol('ADD_MESSAGE');
 
+/**
+ * The type of the action which signals to clear messages in chat on
+ * connecting to conference.
```

Comment:
This jsdoc can probably do without mentioning when it gets called. For example, for spot I want to clear chat messages on conference leave (when trying to switch to a spot).

Reply:
True, I noticed that earlier, just waited for the next push round to fix it. Also changed the time the messages get cleared (see middleware) so it's not more logical.